cmd/adapter-rest/startcmd/start.go

```diff
@@ -1116,3 +1138,25 @@ func getPresentationExchangeProvider(configFile string) (*presentationex.Provide
 
 	return p, nil
 }
+
+func readCMOutputDescriptorFile(outputDescriptorsFile string) (cmOutputDescriptors map[string][]*cm.OutputDescriptor,
```

Comment:
Question: Is each credential scope supposed to map to one output descriptor, or multiple? @sudeshrshetty @talwinder50

Reply:
can be one or multiple its in this format 
“prc_scope1”: [ {outputdescripter1}, {outputdescripter2} ]

Reply:
Are there multiple scopes per credential manifest too? The thing to keep in mind is that when you validate the Output Descriptors, you need to validate all output descriptors that could be grouped within a single Credential Manifest, since one of the requirements from the spec is that each output descriptor (within a single Credential Manifest) has a unique ID.

It's a bit unclear to me how there are multiple output descriptors for one credential scope - I was expecting one descriptor for one scope

Reply:
An issuer can issue multiple credential in a given scope. You can get multiple credentials issued in a single flow by an issuer in single interaction

Reply:
OK, makes sense!